lib/src/layout_overlays.dart

```diff
@@ -124,20 +125,23 @@ class _OverlayBuilderState extends State<OverlayBuilder> {
     super.initState();
 
     if (widget.showOverlay) {
-      WidgetsBinding.instance.addPostFrameCallback((_) => showOverlay());
+      ambiguate(WidgetsBinding.instance)
```

Comment:
I don't see any reason of doing this! Since Flutter 3 is a stable version, we don't have to worry about backward compatibility I think.

Reply:
I guess for the starting phase it's possible that many projects that integrates this plugin might have not migrated to 3.0. So, we should provide backward compatibility for this version at least. Please share your thoughts on this?

Reply:
There's no other major breaking changes in Flutter 3 so people who are using Flutter 2.10 can easily upgrade their projects. And people who haven't upgraded yet can still use the current latest version of the library. Besides most of the packages that I have seen are not keeping backward compatibility.

It's fine if you wanna keep it for this version but it will the same thing for the next version, eventually there's gonna be a version that requires people to upgrade to Flutter 3. Besides this is only helpful in case where you do some bug fixes after the upgrade and people who haven't upgraded yet want to use that fix.

Reply:
Thank you very much for sharing your thoughts. I agree that people can easily migrate from flutter 2 to 3 but still there will be some large projects that does not migrate instantly for safety reasons. While Flutter 2 users can use current latest versions of this package they won't be able to use future updates we provide. 

I still think that we should keep this patch for some time. I agree with you that in future there will be a version that requires people to upgrade to Flutter 3. At that time we can remove this patch I will add TODO for same.

Reply:
Alright! Sounds good enough. You probably don't wanna merge this PR yet.
https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/pull/224